OverlayPlugin.Core/MemoryProcessors/AtkStage/AtkStageMemory62.cs

```diff
@@ -12,10 +12,10 @@ interface IAtkStageMemory62 : IAtkStageMemory { }
 
     class AtkStageMemory62 : AtkStageMemory, IAtkStageMemory62
     {
-        private static int GetAtkStageSingletonAddress(TinyIoCContainer container)
+        private static long GetAtkStageSingletonAddress(TinyIoCContainer container)
         {
             var data = container.Resolve<FFXIVClientStructs.Data>();
-            return (int)data.GetClassInstanceAddress(FFXIVClientStructs.DataNamespace.Global, "Component::GUI::AtkStage");
+            return (long)data.GetClassInstanceAddress(FFXIVClientStructs.DataNamespace.Global, "Component::GUI::AtkStage");
```

Comment:
Should this pass through the `long?` to the caller rather than casting?

Reply:
Currently there's an assumption in the `AtkStageMemory` constructor that it'll receive a valid `long` since this is read from a static file, not calculated based on some runtime state.

Longer-term I think it would be better to have some sort of build-time test to ensure that the requested value exists in the data.yml file, and then remove the error/null checking from `Data.GetClassInstanceAddress`. Maybe I should add a `TODO` comment on `GetClassInstanceAddress` to mention this?

Reply:
That seems reasonable!